lib/internal/test/unicode.js

```diff
@@ -0,0 +1,6 @@
+'use strict';
+
+// This module exists entirely for regression testing purposes.
+// See `test/parallel/test-internal-unicode.js`.
```

Comment:
That doesn't actually matter much if Unicode characters in comments are encoded properly in Node's binary, if that's what you are talking about, IMO, though they may be useful for debugging, [as @Fishrock123 noted](https://github.com/nodejs/node/pull/11417#issuecomment-280363430). It is quite easy to test Unicode in comments automatically, either exporting a function with a comment inside in this module and checking `fn.toString()` in the test, or checking the whole value of `process.binding('natives')['internal/test/unicode']`, but I'd like the status of https://github.com/nodejs/node/pull/11371 and https://github.com/nodejs/node/pull/11417 to become clear before doing this so that this part of the test won't become obsolete as soon as either of these lands (if any). Not to mind that I don't think it is necessary to test Unicode in comments specially at all: how are they different from any other parts of the source code?